front/src/views/profile/accommodation/accommodation.view.html

```diff
@@ -1,38 +1,74 @@
 <div>
   <mmt-stepper></mmt-stepper>
+  <div class="d-flex flex-wrap mt-10" v-if="filters.isPriceChosen || filters.year || filters.countries.length">
+    <h5 class="mr-3">{{ $t('accommodations.filter') }}:</h5>
+    <v-chip v-for="countryId of filters.countries" :key="'country-' + countryId" class="mx-1 mb-3" color="accent" close
+      @click:close="filters.countries = filters.countries.filter(country => country !== countryId ); onFilterUpdate();">
+      {{countries.find(country => country.value === countryId).text}}</v-chip>
+    <v-chip v-if="filters.year" class="mx-1 mb-3" color="accent" close
+      @click:close="filters.year = null; onFilterUpdate();">
+      {{$t('accommodations.year')}}: {{filters.year}}</v-chip>
+    <v-chip v-if="filters.isPriceChosen" class="mx-1 mb-3" color="accent" close
+      @click:close="filters.isPriceChosen = false; onFilterUpdate();">
+      {{filters.price[0]}} $ - {{filters.price[1]}} $ </v-chip>
```

Comment:
maybe you need a dictionary or something similar to a dictionary?

Reply:
dictionary for what ?
slider uses number[2]

Reply:
Oh, I'm sorry, I didn't quite figure out what it was for